### PR TITLE
BACKLOG-11571: improve multi-select input a11y

### PR DESCRIPTION
**Pull request by FBerthelot**
## JIRA

https://jira.jahia.org/browse/BACKLOG-11571

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: src/javascript/DesignSystem/MultipleInput/MultipleInput.jsx [hunk 1]
```diff
@@ -50,6 +48,7 @@ const MultipleInputComponent = ({classes, creatable, readOnly, ...props}) => {
                 styles={classes}
                 isDisabled={readOnly}
                 {...props}
+                isReadOnly={readOnly}
```
Comment (Taybou) [inline src/javascript/DesignSystem/MultipleInput/MultipleInput.jsx]: what's the diff btw isDisabled and isReadOnly?

File: src/javascript/DesignSystem/MultipleInput/components/Control.jsx [hunk 2]
```diff
@@ -1,47 +1,75 @@
...
-            className={`${classes.input}
-                        ${readOnly ? classes.readOnly : ''}`
-            }
-            inputProps={{
-                inputRef: ref,
-                children: children,
-                ...innerProps
-            }}
-        />
+        <div
```
Comment (Taybou) [inline src/javascript/DesignSystem/MultipleInput/components/Control.jsx]: div, encore! okay :3

File: src/javascript/DesignSystem/MultipleInput/MultipleInput.jsx (hunk 1 above)
Comment (FBerthelot) [reply src/javascript/DesignSystem/MultipleInput/MultipleInput.jsx]: Ask to designer for the answer... It was the major pain point on this pull request :/

File: src/javascript/DesignSystem/MultipleInput/components/Control.jsx (hunk 2 above)
Comment (FBerthelot) [reply src/javascript/DesignSystem/MultipleInput/components/Control.jsx]: Control component is a wrapper for many components in react-select. The real input is in children ;)

**Pull request merged**